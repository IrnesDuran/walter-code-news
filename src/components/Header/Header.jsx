import React from 'react';

const Header = () => {
    return(
        <div className="bg-primary p-4 fixed w-full z-10">
            <a href="/">
                <img src="https://waltercode.com/wp-content/uploads/2019/11/logo-min.png" alt="WalterCode Logo"/>
            </a>
        </div>
    )};

export default Header;
