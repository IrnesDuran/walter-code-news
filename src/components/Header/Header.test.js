import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

import Header from './Header';


describe ("Header component test",()=> {

    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<Header/>);
    });
    

    it('should test Header component snapshot',()=>{
        expect(wrapper).toMatchSnapshot();
    });
})