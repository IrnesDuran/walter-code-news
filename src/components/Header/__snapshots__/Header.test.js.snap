// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component test should test Header component snapshot 1`] = `ShallowWrapper {}`;
